lms/templates/ccx/underscore/unit.underscore

```diff
@@ -0,0 +1,58 @@
+<td class="unit">
+  <% if (unit.category !== "vertical") { %>
+    <button class="toggle-collapse ccx-button-link" aria-expanded="false">
+      <i class="fa fa-caret-right"></i>
+      <span class="sr">
+        <%- interpolate(gettext('toggle chapter %(displayName)s'),
```

Comment:
Just some information-- you can now use StringUtils, which has a somewhat nicer signature for interpolate.